### ISSUE TITLE
Create a KeeWeb Docker image for ARM arch

### ISSUE THREAD

**Issue by nagromc** [keeweb/keeweb#642]
It would be great to have the KeeWeb Docker image support for ARM architectures.

The [official image](https://hub.docker.com/r/antelle/keeweb/) is based on [nginx](https://hub.docker.com/_/nginx/):[stable](https://github.com/nginxinc/docker-nginx/blob/4d1f7f8ec281117d1d79bed4c6bc28b86039ca84/stable/stretch/Dockerfile), which is based on [debian](https://hub.docker.com/_/debian/):[stretch-slim
](https://github.com/tianon/docker-brew-debian/blob/6a620dbc1df446fcabdcd4c9c42a650a26377eb5/stretch/slim/Dockerfile)

Docker released [official images for ARM architectures](https://github.com/docker-library/official-images#architectures-other-than-amd64), especially [arm32v6/alpine](https://hub.docker.com/r/arm32v6/alpine/) and [arm64v8/alpine](https://hub.docker.com/r/arm64v8/alpine/) where ARMv6 is the architecture used on the Raspberry Pi 1, and the ARMv8 on the Raspberry Pi 3. Maybe you should consider to switch to an Alpine-based Nginx Docker image (which should be [lighter](https://github.com/docker-library/docs/tree/master/nginx#nginxalpine) than a [Debian](https://github.com/nginxinc/docker-nginx/blob/4d1f7f8ec281117d1d79bed4c6bc28b86039ca84/stable/stretch/Dockerfile)-based image BTW).

An [issue on the Nginx official repo](https://github.com/nginxinc/docker-nginx/issues/99) already exists, and I [contacted](https://github.com/nginxinc/docker-nginx/issues/99#issuecomment-304334331) the author to let them know the ARM-based images are available.

**Comment by antelle** [comment]
But nginx-alpine is not compatible with arm, right?
We can switch to it, once it's available, I don't see any obvious reasons why we should not. As we're not using binary components, it should be fine.

**Comment by nagromc** [comment]
Yes, the Docker image [nginx:alpine](https://hub.docker.com/_/nginx/), relies on [alpine:3.5](https://hub.docker.com/_/alpine/), which — I think — is not compatible with ARM architectures. We are waiting for https://github.com/nginxinc/docker-nginx/issues/99 to be resolved.

Thanks for you amazing work.

**Comment by jochec** [comment]
Seems like ARM is supported now - [https://hub.docker.com/_/nginx/](https://hub.docker.com/_/nginx/)

**Comment by antelle** [comment]
Good, then we can build an image with it

**Comment by fmnamado** [comment]
Hello,
I am looking for a raspberry pi 4 official Kee Web image but I can't seem to found it.
I am talking about ARM64
Has this ever been done?
The text above seems that at least there was an intention in the past.

Thank you!
